sonic-xcvrd/xcvrd/dom/dom_mgr.py

```diff
@@ -198,6 +198,47 @@ def post_port_pm_info_to_db(self, logical_port_name, port_mapping, table, stop_e
             else:
                 return xcvrd.SFP_EEPROM_NOT_READY
 
+    def wait_port_initialization(self, delay):
+        logical_port_set = set(self.port_mapping.logical_port_list)
+
+        while logical_port_set:
```

Comment:
@prgeor Can we add an upper bound timeout here to ensure that we don't end up in infinite loop (ideally, this should not occur)?